mods/classic-desktop-icons.wh.cpp

```diff
@@ -105,6 +119,61 @@ HWND FindDesktopWindow(void)
     return NULL;
 }
 
+
+
+using CreateWindowExW_t = decltype(&CreateWindowExW);
+CreateWindowExW_t CreateWindowExW_orig;
+HWND CreateWindowExW_hook(
+    DWORD     dwExStyle,
+    LPCWSTR   lpClassName,
+    LPCWSTR   lpWindowName,
+    DWORD     dwStyle,
+    int       X,
+    int       Y,
+    int       nWidth,
+    int       nHeight,
+    HWND      hWndParent,
+    HMENU     hMenu,
+    HINSTANCE hInstance,
+    LPVOID    lpParam
+)
```

Comment:
You're missing a calling convention (WINAPI), which will likely cause a crash in 32-bit. See existing mods that hook `CreateWindowExW`.